src/page-modules/contact/means-of-transport/means-of-transport-form-selector.tsx

```diff
@@ -0,0 +1,63 @@
+import { PageText, useTranslation } from '@atb/translations';
+import { SectionCard } from '../components/section-card';
+import { RadioInput } from '../components/input/radio';
+import { ContextProps } from './means-of-transport-form-machine';
+import { machineEvents } from '../machineEvents';
+
+type MeansOfTransportFormSelectorProps = {
+  state: {
+    hasTag(arg0: string): boolean | undefined;
+    context: ContextProps;
+  };
+  send: (event: typeof machineEvents) => void;
+};
+
+export const MeansOfTransportFormSelector = ({
+  state,
+  send,
+}: MeansOfTransportFormSelectorProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <SectionCard title={PageText.Contact.travelGuarantee.title}>
+      <ul>
+        <RadioInput
+          label={t(PageText.Contact.modeOfTransport.driverForm.description)}
+          checked={state.hasTag('driverForm')}
+          onChange={() => send({ type: 'DRIVER_FORM' })}
+        />
```

Comment:
Instead of using `checked={state.hasTag('driverForm')}`, would it be more correct to set the value to `DRIVER_FORM` and let the onChange use the `UPDATE_FIELD` event instead? I believe this is the more correct way of using radio inputs.